contracts/TokenConversionLib.sol

```diff
@@ -0,0 +1,57 @@
+pragma solidity 0.4.23;
+
+import "./SafeMath.sol";
+
+library TokenConversionLib {
+    using SafeMath for uint256;
+
+    /**
+     *	@notice Calculate Utility token amount based on value token amount, conversion rate and conversion decimal
```

Comment:
Please make "utility" in "Calculate Utility token..." lowercase. Please make sure you end all sentences with a period as per our style guide.